python/plugins/processing/algs/qgis/RasterCalculator.py

```diff
@@ -197,7 +197,7 @@ def processBeforeAddingToModeler(self, algorithm, model):
         for i in list(model.inputs.values()):
             param = i.param
             if isinstance(param, QgsProcessingParameterRasterLayer) and "{}@".format(param.name) in expression:
-                values.append(ValueFromInput(param.name()))
+                values.append(ValueFromInput(param.name()))  # noqa TODO: how is that supposed to work??
```

Comment:
@nyalldawson do you have an idea what "ValueFromInput" is supposed to call or do? It seems to originate in some ModellerAlgorithm python module that no longer exists. This is the only leftover from this.

Reply:
Kill it. processBeforeAddingToModeler is not used anywhere